smarty/templates/email/permissions_change_notify_supervisor.tpl

```diff
@@ -0,0 +1,12 @@
+Subject: Permission Changes for {$realname} - {$study}
+
+The permissions have been changed for {$realname} (user name: {$username}).
+
+The following permissions have changed:
+{foreach from=$changedpermissions item=foo}
+    - {$foo}
+{/foreach}
+
+Thank you,
+
+LORIS Team
```

Comment:
How about adding more info/context to the email? Something like "You are receiving this email because user {xyz} changed the permissions of user {abc} on Loris tool {url}...." Maybe Leigh would have some suggestions on the content?